bucket/keepass-plugin-keepassrpc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://www.kee.pm/",
+    "description": "Kee is a Firefox and Chrome add-on for linking browsers to KeePass, using the KeePassRPC KeePass plugin contained within this repository.",
+    "license": "GPL-2.0-or-later",
```

Comment:
Also not sure about the license. It's certainly is [GPL2.0 ](https://github.com/kee-org/keepassrpc/blob/v1.8.0/LICENSE), but I don't if it's "2.0 and later" or "2.0 only".

Reply:
-or-later is fine.